### DISCUSSION TITLE
What will be the latest version of tinymce-react that supports the latest tinymce ^6.0.0?

### DISCUSSION THREAD

**Discussion by dqhendricks** [tinymce/tinymce-react#539] [Q&A] [answered]
I do not want to move to TinyMCEs new version 7 licensing, and so want to make sure my project only ever uses whatever the latest version of tinymce version 6 (never 7+). For this purpose I would like to know how I should set up my tinymce-react versioning to be in my project. Any guidance would be greatly appreciated. Cheers!

**Comment by danoaky-tiny** [comment, marked as answer]
You can use any version of TinyMCE you want with tinymce-react. But the default props are set up for using the Tiny Cloud CDN and whatever the newest version of TinyMCE is at the time.

To load version 6 instead, you have a couple of options:
- Set the `cloudChannel` prop to `6` along with an `apiKey` to use Tiny Cloud hosting, see: https://www.tiny.cloud/docs/tinymce/latest/react-cloud/
- [Package manager hosting with placing tinymce in public/static folder](https://www.tiny.cloud/docs/tinymce/latest/react-pm-host/)
- [Package manager hosting with bundling](https://www.tiny.cloud/docs/tinymce/latest/react-pm-bundle/)
- [Zip file hosting with placing tinymce in public/static folder](https://www.tiny.cloud/docs/tinymce/latest/react-zip-host/)
- [Zip file hosting with bundling](https://www.tiny.cloud/docs/tinymce/latest/react-zip-bundle/)

Let us know if you need further clarification on any of these.

**Reply by dqhendricks** [reply]
Thank you!

**Reply by dqhendricks** [reply]
Unfortunately it seems that the link for "Zip file hosting with placing tinymce in public/static folder" is broken.

**Reply by danoaky-tiny** [reply]
Ah apologies, this has been escalated internally.
